city_scrapers/spiders/det_employment_solutions.py

```diff
@@ -0,0 +1,184 @@
+         - DESC Meetings
+        """
+        elementor_row = response.css(
+            "section.elementor-element:nth-child(4)"
+            + "> div:nth-child(1) > div:nth-child(1)"
+        )
+        # The css selector for the left section of the page
+        meeting_sections = elementor_row.css("section.desc-public-meeting")
+        # Assumes location is the same across meeting types.
+        location = self._parse_location(meeting_sections[0])
+        # Appends the css selector for the right section on the page
+        meeting_sections.append(
+            elementor_row.css("div.elementor-row > div.elementor-element-77d4b08")
+        )
+
+        for items in meeting_sections[1:]:
+            meeting_type = self.get_meeting_type(items)
+            for meeting_item in items.css(
+                self.MEETING_TYPE_TO_CSS_SELECTOR_MAP[meeting_type]
+            ):
+                meeting = Meeting(
+                    title=self._parse_title(items, meeting_type),
+                    description="",
+                    classification=self._parse_classification(items, meeting_type),
+                    start=self._parse_start(meeting_item, meeting_type),
+                    end=self._parse_end(meeting_item, meeting_type),
+                    all_day=False,
+                    time_notes=self._parse_time_notes(items),
+                    location=location,
+                    links=self._parse_links(meeting_item, meeting_type),
+                    source=response.url,
+                )
+                meeting["id"] = self._get_id(meeting)
+                meeting["status"] = self._get_status(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item, meeting_type):
+        """Parse or generate meeting title"""
+        if meeting_type == "MWDB":
+            return "Mayor's Workforce Development Board (MWDB) Meeting"
+        if meeting_type == "CEAC":
+            return "Career and Education Advisory Council (CEAC) Meeting"
+        return "DESC Meeting"
+
+    def _parse_classification(self, item, meeting_type):
+        """Parse or generate classification from allowed options."""
+        if meeting_type == "MWDB":
+            return BOARD
+        if meeting_type == "CEAC":
+            return ADVISORY_COMMITTEE
+        if meeting_type == "DESC":
+            meeting_name = item.css("strong ::text").get()
+            if meeting_name == "Corporation Board Meeting":
+                return BOARD
+            if meeting_name == "Executive Committee Meeting":
+                return COMMITTEE
+
+        return BOARD
+
+    def find_date(self, item, meeting_type, is_start=False):
+        """Parses item and creates a datetime formatted string."""
+        if meeting_type is None:
+            return None
+        if meeting_type == "MWDB":
+            # remove the 'nd' 'rd' 'th' from number day
+            caldate = item.css("strong ::text").get().strip()[:-2]
```

Comment:
We've seen a few agencies include the incorrect day of the week but correct date, so since it doesn't add anything it's probably safest to ignore

Reply:
Just want to double check. Would you like me to remove the finding `caldate` and just get the time?